docs/source/get-involved/gsoc/projects_2025/brainglobe.md

```diff
@@ -45,9 +46,95 @@ Experience with Python, [NumPy](https://numpy.org/doc/stable/index.html) and/or
+
+
+**Deliverables**
+<!-- Goals, or expected status after Community Bonding Period, Start of Coding, End of Coding. Stretch goals? -->
+- A Python implementation of a signal processing algorithm (blob detection) in `cellfinder` that detects cell candidates in two-dimensional images.
+- A Python implementation of a neural network in `cellfinder` that classifies cell candidates from two-dimensional images.
+- Tests to cover any added functionality.
+- Documentation for the new functionality.
+- A [blog](https://brainglobe.info/blog) showcasing the use of `cellfinder` on two-dimensional images of brains.
+
+**Duration**
+<!-- Small (~90 hours), Medium (~175 hours) or Large (~350 hours)  -->
+Large (~350 hours).
+
+**Difficulty**
+<!-- Is this project geared more toward a student level or a more advanced developer level? -->
+This project is well-suited for an intermediate contributor to open source.
+
+**Required skills**
+
+- Experience with Python, [NumPy](https://numpy.org/doc/stable/index.html) and/or [pandas](https://pandas.pydata.org/docs/index.html).
+- At least initial understanding of machine learning algorithms
+
+**Nice-to-haves**
+- Experience working with machine learning frameworks, in particular `keras` or `pytorch`
+- Experience working with image data
+
+
+**Potential mentors**
+- [@IgorTatarnikov](https://github.com/IgorTatarnikov)
+- [@alessandrofelder](https://github.com/alessandrofelder)
+
+
+**Further reading**
+<!-- The best pages include links to more detailed descriptions and related materials for each project. They might even include actual use cases! -->
+- [BrainGlobe developer guide](https://brainglobe.info/community/developers/index.html)
+- [cellfinder paper](https://doi.org/10.1371/journal.pcbi.1009074)
+- [cellfinder code](https://github.com/brainglobe/cellfinder)
+:::
+
+:::{dropdown} {fas}`brain;sd-text-primary` `cellfinder` support for arbitrary number of channels
+<!-- Description -->
+The [BrainGlobe `cellfinder` tool](https://brainglobe.info/documentation/cellfinder/index.html) is used to detect cells in large whole-brain images. It uses signal processing to find possible cell candidates and passes them to a customisable classifier to split the candidates into cells and no-cells. `cellfinder` relies heavily on `pytorch` and `keras`.
+
+`cellfinder` was originally designed to work with two channels: the "background" channel and the "signal" channel. This is a limitation, because sometimes only the signal channel is acquired, or there are several signal channels containing useful information. In this project, you would explore ways to allow `cellfinder` to classify cells based on information from an arbitrary number of channels.
```

Comment:
```suggestion
`cellfinder` was originally designed to work with two channels: the "background" channel and the "signal" channel. This is a limitation, because sometimes only the signal channel is acquired, or there are several signal channels containing useful information. In this project, you would explore ways to allow `cellfinder` to classify cells based on information from an arbitrary number of channels. This could allow cellfinder to not just detect cells, but classify their type based on their shape and other features. 
```

Reply:
Nice!